File: snippets/single-file-single-thread/array-methods/javascript/basic.js
let numbers = [1, 2, 3];

numbers.push(4);           // [1, 2, 3, 4]
numbers.unshift(0);        // [0, 1, 2, 3, 4]
numbers.pop();             // removes 4, returns it
numbers.shift();           // removes 0, returns it
const length = numbers.length;  // 3
numbers.sort((a, b) => a - b);  // sorts numerically
numbers.reverse();         // reverses in place

File: snippets/single-file-single-thread/array-methods/python/basic.py
numbers = [1, 2, 3]

numbers.append(4)          # [1, 2, 3, 4]
numbers.insert(0, 0)       # [0, 1, 2, 3, 4]
numbers.pop()              # removes 4, returns it
numbers.remove(2)          # removes first occurrence of 2
length = len(numbers)      # 3
numbers.sort()             # sorts in place
numbers.reverse()          # reverses in place
